app/controllers/application_controller.rb

```diff
@@ -26,66 +26,27 @@ def document_type
   end
 
   def current_format
-    document_types.fetch(params.fetch(:document_type, nil), nil)
+    document_types.detect { |format| format.document_type == document_type }
   end
```

Comment:
Old `current_format` impl. was a hash lookup so not much to be gained by memoisation, but now it's iterating through an array to find something, so there will be tiny benefits to be gained for all requests for `VehicleRecallsAndFaultsAlert` documents ;)